homeassistant/components/goalzero/binary_sensor.py

```diff
@@ -0,0 +1,66 @@
+"""Support for Goal Zero Yeti Sensors."""
+import logging
+
+from homeassistant.components.binary_sensor import BinarySensorEntity
+from homeassistant.const import CONF_NAME
+
+from . import YetiEntity
+from .const import (
+    BINARY_SENSOR_DICT,
+    BINARY_SENSOR_LIST,
+    DATA_KEY_API,
+    DATA_KEY_COORDINATOR,
+    DOMAIN,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+PARALLEL_UPDATES = 0
+
+
+async def async_setup_entry(hass, entry, async_add_entities):
+    """Set up the Pi-hole sensor."""
```

Comment:
This ain't Pi-hole 😉

Reply:
Oops, was afraid I'd do that. You know the whole story then 😉